src/test/scala/cromwell/binding/ScatterSpec.scala

```diff
@@ -0,0 +1,62 @@
+package cromwell.binding
+
+import cromwell.parser.BackendType
+import cromwell.util.SampleWdl
+import org.scalatest.{FlatSpec, Matchers}
+
+class ScatterSpec extends FlatSpec with Matchers {
+  val namespace = NamespaceWithWorkflow.load(SampleWdl.ScatterWdl.wdlSource(), BackendType.LOCAL)
+
+  it should "Have four 'children' objects" in {
+    namespace.workflow.children.size shouldEqual 4
+  }
+
+  it should "Have two 'direct Call descendents' objects" in {
+    Scope.collectCalls(namespace.workflow.children).size shouldEqual 2
+  }
+
+  it should "Have two 'direct Scatter descendents' objects" in {
+    Scope.collectScatters(namespace.workflow.children).size shouldEqual 2
+  }
+
+  it should "Have seven 'Call' objects" in {
+    namespace.workflow.calls.size shouldEqual 8
+  }
+
+  it should "Have three 'Scatter' objects" in {
+    namespace.workflow.scatters.size shouldEqual 4
+  }
+
+  it should "Have 'Scatter' objects indexed properly" in {
+    namespace.workflow.scatters.zipWithIndex foreach {
+      case (s,i) => s.idx shouldEqual i
+    }
+  }
+
+  it should "Not let scatters appear in FQNs" in {
+    namespace.workflow.calls foreach { c =>
+      c.fullyQualifiedName shouldNot include regex "\\$scatter_[0-9]+"
+    }
+
+    namespace.workflow.declarations foreach { d =>
+      d.fullyQualifiedName shouldNot include regex "\\$scatter_[0-9]+"
+    }
```

Comment:
I think it might be more valuable in these tests to assert that the FQNs are what you expect them to be for each 'call'